engine/src/main/java/org/hibernate/validator/internal/metadata/location/ConstraintLocation.java

```diff
@@ -104,14 +104,6 @@ static ConstraintLocation forParameter(Callable callable, int index) {
 	 */
 	ConstraintLocationKind getKind();
 
-	interface Builder {
-		ConstraintLocation build(Constrainable constrainable);
-
-		static Builder forPropertyHolderProperty() {
-			return new PropertyHolderPropertyConstraintLocationBuilder();
-		}
-	}
-
```

Comment:
Maybe it should be removed in the commit where it has been added? Or was it useful at the time?